wiki/en/Include-QOS-Windows.md

```diff
@@ -3,13 +3,13 @@
 
 # Quality of Service (QoS)
 
-Jamulus uses DSCP/CS4 opportunistically to deal with buffer bloat, and uses a default DSCP/CS4 value of 128 (or 0x80). This is compatible with IPv4 and IPv6. Other values can be set using the -Q option, eg -Q [0..255] (where 0 disables QoS). If you want to explore the effect of non-default settings, see RFC4594. However, most people will have no need to do this.
+Jamulus uses DSCP/CS4 to opportunistically deal with buffer bloat. Setting the DS field byte to 128 (or 0x80) in a IPv4 or IPv6 packet header selects DSCP/CS4. Other byte values can be set using the -Q option, e.g. -Q [0..255] (where 0 cripples QoS). If you want to explore the effect of non-default settings, see RFC4594. However, most people will have no need to do this.
 
 ## Use of QoS on Windows
 
-Jamulus's QoS settings (including the default) have no effect on Windows because the operating system ignores them. To enable it, you must follow these instructions. Note also that you may need to repeat this procedure every time Jamulus is updated.
+Jamulus' QoS settings (including the default) are set to 0 on recent Windows and Apple because of the operating system. To use QoS on Windows, follow these instructions. Note that you may need to repeat this procedure every time Jamulus is updated.
```

Comment:
```suggestion
Jamulus' QoS settings (including the default) are set to 0 on recent Windows and macOS because of the operating system. To use QoS on Windows, follow these instructions. Note that you may need to repeat this procedure every time Jamulus is updated.
```
Is macOS now also un-changable? Why did you add it here?

Reply:
Darwin and later kernels were changed to operate in an undocumented? macOS manner.
https://github.com/jamulussoftware/jamulus/discussions/1988#discussion-3551297

Windows did something similar, but left just enough for a workaround.